tests/functional/oauth_webchannel.js

```diff
@@ -2,6 +2,9 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+// TODO: remove once https://github.com/eslint/eslint/issues/1801 gets fixed
+/* eslint indent: 0 */
+
```

Comment:
Workaround:
```
        }, function () {
          // element was not found
        }) /* HACK: See eslint/eslint#1801 */ // eslint-disable-line indent
```

I think you only need that in 4 places in this file. So it'd be 4 line diffs vs the 2-3 above.